pkg/services/ngalert/schedule/jitter.go

```diff
@@ -0,0 +1,68 @@
+package schedule
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/prometheus/prometheus/model/labels"
+)
+
+// JitterStrategy represents a modifier to alert rule timing that affects how evaluations are distributed.
+type JitterStrategy int
+
+const (
+	JitterNever JitterStrategy = iota
+	JitterByGroup
+	JitterByRule
+)
+
+// JitterStrategyFromToggles returns the JitterStrategy indicated by the current Grafana feature toggles.
+func JitterStrategyFromToggles(toggles featuremgmt.FeatureToggles) JitterStrategy {
```

Comment:
```suggestion
func JitterStrategyFrom(toggles featuremgmt.FeatureToggles) JitterStrategy {
```

nit: this typically stutters - I'd favor `schedule.JitterStrategyFrom(ng.FeatureToggles)`

Reply:
This flows better, thanks!